ingest/de.py

```diff
@@ -171,6 +171,12 @@ def execute_de(self):
                 self.method,
             )
             DifferentialExpression.de_logger.info("preparing DE on dense matrix")
+        else:
+            msg = f"Submitted matrix_file_type should be \"dense\" or \"mtx\" not {self.matrix_file_type}"
```

Comment:
Slightly easier readability:

```suggestion
            msg = f"Submitted matrix_file_type should be \"dense\" or \"mtx\" not \"{self.matrix_file_type}\""
```